src/components/crossDevice/CrossDeviceLink/index.js

```diff
@@ -103,22 +103,33 @@ class CrossDeviceLinkUI extends Component {
       copySuccess: false,
       sending: false,
       error: {},
-      validNumber: true,
+      validNumber: true
     }
   }
 
   linkId = `${process.env.BASE_32_VERSION}${this.props.roomId}`
 
   linkCopiedTimeoutId = null
 
-  copyToClipboard = (e) => {
-    this.linkText.select()
-    document.execCommand('copy')
-    e.target.focus()
+  copyToClipboard = (mobileUrl) => {
+    let tempInput = document.createElement('input')
+    document.body.appendChild(tempInput);
```

Comment:
Why do you need a temporary input?

Reply:
So that the URL text can be selected from the temp input in document to be copied to the clipboard. I was not able to get the URL text copied to the clipboard from a non-input element. 
This also avoids the scenario we saw where the screen reader's focus kept skipping to the visible textarea element with the URL on clicking the Copy button.